docs/dispersion/index.rst

```diff
@@ -2,21 +2,25 @@
 Dispersion (`plasmapy.dispersion`)
 **********************************
 
+.. py:currentmodule:: plasmapy.dispersion
+
 .. automodule:: plasmapy.dispersion
 
 Example Notebooks
 -----------------
 
 .. nbgallery::
 
-   /notebooks/dispersion_function
+   /notebooks/dispersion/dispersion_function
+   /notebooks/dispersion/two_fluid_dispersion
```

Comment:
```python
.. nbgallery::
   :glob:
   :reversed:

   /notebooks/dispersion/*
```

This will cause the thumbnail gallery to automatically update whenever a new notebook is added.  And, I definitely see more notebooks being added down the road.